src/init.ts

```diff
@@ -193,19 +202,10 @@ export class NightwatchInit {
     }
 
     if (answers.addExamples && !answers.examplesLocation) {
-      if (answers.runner === 'cucumber') {
-        answers.examplesLocation = path.join(answers.featurePath || '', 'nightwatch-examples');
+      if (answers.runner === Runner.Cucumber) {
+        answers.examplesLocation = path.join(answers.featurePath || '', DEFAULT_FOLDER);
       } else {
-        // Find a location for putting the example files.
-        const testsDestPath = path.join(this.rootDir, answers.testsLocation);
-        if (fs.existsSync(testsDestPath) && fs.readdirSync(testsDestPath).length) {
-          // If testsLocation already contains some files, put the examples in a
-          // separate directory.
-          answers.examplesLocation = 'nightwatch-examples';
-        } else {
-          // Put examples directly into testsLocation, to be used as boilerplate.
-          answers.examplesLocation = answers.testsLocation;
-        }
+        answers.examplesLocation = DEFAULT_FOLDER;
```

Comment:
Since the major motive for adding the changes removed here was to allow/encourage new users to use our examples as boilerplate (start writing their own tests over our example tests or just use the folder structure provided by us) by putting the examples directly in their tests directory (if empty), I'm more inclined towards keeping it. Maybe just replace `'nightwatch-examples'` with `DEFAULT_FOLDER`?

/cc: @beatfactor since the previous changes were added based on your suggestion.